tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -309,7 +316,7 @@ fn build_nsis_app_installer(
   let main_binary_path = settings.binary_path(main_binary).with_extension("exe");
   data.insert(
     "main_binary_name",
-    to_json(main_binary.name().replace(".exe", "")),
+    to_json(main_binary.name().strip_suffix(".exe")),
```

Comment:
Let's add a fallback, just in case it doesn't end with `.exe`
```suggestion
    to_json(main_binary.name().strip_suffix(".exe").unwrap_or(main_binary.name())),
```